trengerservice/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/trengerservice/TrengerService.kt

```diff
@@ -191,42 +202,69 @@ class TrengerService(
                 redisStore.set(RedisKey.of(clientId), TrengerData(feilReport = feilReport).toJsonStr(TrengerData.serializer()))
             }
             return
-        } else if (utloesendeBehov == BehovType.VIRKSOMHET) {
-            feilmelding = Feilmelding("Vi klarte ikke å hente virksomhet navn.", datafelt = Key.VIRKSOMHET)
-            redisStore.set(RedisKey.of(fail.transaksjonId, Key.VIRKSOMHET), "Ukjent navn")
-        } else if (utloesendeBehov == BehovType.FULLT_NAVN) {
-            feilmelding = Feilmelding("Vi klarte ikke å hente arbeidstaker informasjon.", datafelt = Key.ARBEIDSTAKER_INFORMASJON)
-            redisStore.set(
-                RedisKey.of(fail.transaksjonId, Key.ARBEIDSTAKER_INFORMASJON),
-                PersonDato("Ukjent navn", null, "").toJsonStr(PersonDato.serializer())
-            )
-            redisStore.set(
-                RedisKey.of(fail.transaksjonId, Key.ARBEIDSGIVER_INFORMASJON),
-                PersonDato("Ukjent navn", null, "").toJsonStr(PersonDato.serializer())
-            )
-        } else if (utloesendeBehov == BehovType.INNTEKT) {
-            feilmelding = Feilmelding(
-                "Vi har problemer med å hente inntektsopplysninger. Du kan legge inn beregnet månedsinntekt manuelt, eller prøv igjen senere.",
-                datafelt = Key.INNTEKT
-            )
-            redisStore.set(RedisKey.of(fail.transaksjonId, Key.INNTEKT), UNDEFINED_FELT)
         }
-        if (feilmelding != null) {
-            val feilKey = RedisKey.of(fail.transaksjonId, feilmelding)
-            val feilReport: FeilReport = redisStore.get(feilKey)?.fromJson(FeilReport.serializer()) ?: FeilReport()
-            feilReport.feil.add(feilmelding)
+
+        val datafeil = when (utloesendeBehov) {
```

Comment:
Vi kommer jo fortsatt til å bare miste orgnr / fnr dersom oppslag feiler..

Reply:
Orgnr mister vi ikke: https://github.com/navikt/helsearbeidsgiver-inntektsmelding/pull/402/files#diff-58067f1e20fdffc484346e6c48099735aad70f03865ad0932c0dc06e19487e8cR169.

For fnr så har det ingen betydning, da de ikke blir brukt til noe. Det er bare navnet som sendes videre til frontend.